### PR TITLE
values.yaml: add more detail around using connectCA with vault namespaces

### PR DESCRIPTION
**Pull request by david-yu**
Changes from corresponding docs PR: https://github.com/hashicorp/consul/pull/14221/files

Changes proposed in this PR:
-
-

How I've tested this PR:

How I expect reviewers to test this PR:

Checklist:
- [ ] Tests added
- [ ] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

**Comment by david-yu** [comment]
cc @jkirschner-hashicorp

**Review by kschoche — APPROVED** [review]
Thanks for taking care of this!

File: charts/consul/values.yaml [hunk 1]
```diff
@@ -271,16 +271,16 @@ global:
         authMethodPath: "kubernetes"
 
         # The path to a PKI secrets engine for the root certificate.
-        # Please see https://www.consul.io/docs/connect/ca/vault#rootpkipath.
+        # For more details, please refer to [Vault Connect CA configuration](https://www.consul.io/docs/connect/ca/vault##rootpkipath).
```
Comment (jkirschner-hashicorp) [inline charts/consul/values.yaml:274]: Small error. I'll make sure it's not present in my PR.

```suggestion
        # For more details, please refer to [Vault Connect CA configuration](https://www.consul.io/docs/connect/ca/vault#rootpkipath).
```

**Review by jkirschner-hashicorp — CHANGES_REQUESTED** [review]
One nit

**Comment by jkirschner-hashicorp** [comment]
Is https://github.com/hashicorp/consul/pull/14221 still needed after this PR is merged? Thanks @david-yu !

**Review by ishustava — APPROVED** [review]
LGTM

**Comment by david-yu** [comment]
@jkirschner-hashicorp no need to merge the docs PR, if this is merged, it will show up though when the next release is out.

**Pull request merged**